### PR TITLE
Goodbye mini-talk

### PR DESCRIPTION
**Pull request by Dormouse759**

### CODE REVIEW

File: series/brno-pyvo/events/2019-08-29.yaml [hunk 1]
```diff
@@ -15,4 +15,8 @@ description: |
     we will not experience pub sauna as we did the last time.
 
 talks:
-    []
+  - title: Goodbye
+    description: |
+        ???
+    speakers: |
+        sha256:fd9f3f0f8e07d220a653add9f7dc9e052a3b232af0ca8c2eedb7aeeb5d62982b
```
Comment (hroncok) [inline series/brno-pyvo/events/2019-08-29.yaml]: why?

Comment (Dormouse759) [reply series/brno-pyvo/events/2019-08-29.yaml]: Thought it's a nice detail. Some personal "signature".
Any idea how to make it not rise the syntax error?

Comment (hroncok) [reply series/brno-pyvo/events/2019-08-29.yaml]: Sure, this is YAML and this string contains `:`, so you need to [put in in quotes](http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html).

Comment (encukou) [reply series/brno-pyvo/events/2019-08-29.yaml]: The actual issue is that `speakers` is a list. You need to put a `- ` in front.

**Pull request merged**